README.md

```diff
@@ -1,364 +1,355 @@
+If there is no nomination at all, the Annual General Meeting shall resolve this matter.  
+### 7.6. Voting
+Voting shall be held at week 1–4 of the spring semester, within 7 clear days after the end of the promotion period. Notice of Voting shall be posted on School of Engineering notice areas at least 7 clear days beforehand.
+#### 7.6.1. Eligibility
+Only Full Members of the Society shall have the right to vote. Voting shall be conducted through General Polling in straight and in accordance with the regulations of General Polling.
+#### 7.6.2. Validity
+The result of the Polling is valid provided that the number of votes is not less than twenty-five percent of the population of Full Members of the Society.
+#### 7.6.3. Success
+The cabinet is successful upon
+- If there is only one cabinet running for the Election, the cabinet is successful upon receiving a simple majority of confidence over non-confidence votes; or
+- If there are two or more cabinets running for the Election, the successful cabinet will be the one receiving the greatest number of votes.
+#### 7.6.4. Announcement
+The provisional and official result of the Election shall be announced by the Election Board within 24 and 72 hours after the closure of the voting respectively.
+### 7.7. Re-election
+If the Election fails to generate a new Executive Committee, a re-election shall be conducted. If the re-election also fails to generate a new Executive Committee, a second re-election shall be conducted.  
+If the second re-election also fails to generate a new Executive Committee, an Extraordinary General Meeting shall resolve the matter. Re-election, if any, shall be held before the position of the following Annual General Meeting.  
+The Re-election shall follow the rules and regulations of the Election.  
+### 7.8. Consultation
+The Consultation shall be convened by the President of the Society and shall be held within 1 week once after the Nominated Cabinet is being elected.  
+Five percent or twenty (whichever is greater) of Full Members or General Members of the Society shall form a quorum of the Consultation. If the quorum has not been met, then the Consultation is failed.  
+#### 7.8.1. Agenda
+The business of the Consultation shall be
+1. To receive and adopt or amend the proposed Year Plan prepared by the Executive Committee of the next session.
+2. To receive and adopt or amend the proposed Financial Budget prepared by the Financial Secretary of the Executive Committee of the next session.
+#### 7.8.2. Postponement
+A Consultation shall be postponed if the President of the Society is absent or the quorum is not reached provided that there is a thirty minutes of waiting period. Another Consultation shall be convened within 7 days after the failure of the previous one.
```

Comment:
there is a thirty minutes of waiting period -> there is a waiting period of thirty minutes